manifests/custom_config.pp

```diff
@@ -115,9 +115,10 @@
       require     => Anchor['::apache::modules_set_up'],
     }
 
+    $remove_command = ['/bin/rm', join([$confdir, $_filename], '/')]
```

Comment:
Given that the same concat happens on line 97, should a dedicated variable be introduced for the filename?

And a whole different thought: `file` has a `validate_cmd` option. However, I don't think it can be used in this case since you can't validate the config file by itself. It needs the whole apache config. That's probably why this rollback mechanism was invented here. It does make you wonder if the `file` type should have a `validate_mode` where it can work in a mode where it replaces the real file rather than a temporary file and rolls back if needed.

Reply:
To the first point, it does look better and removes the chance of them getting out of sync, so have implemented.

As to the second.... it sounds like a reasonable idea to me.
Could be something to raise.